data/json/vehicleparts/vehicle_parts.json

```diff
@@ -320,6 +320,30 @@
     "breaks_into": [ { "item": "splinter", "count": [ 7, 9 ] }, { "item": "nail", "charges": [ 5, 10 ] } ],
     "damage_reduction": { "all": 16 }
   },
+  {
+    "type": "vehicle_part",
+    "id": "cloth_aisle",
```

Comment:
How would a cloth floor work?

Reply:
Probably not super well for actually walking around on, admittedly. I figured if the foldable frame's supports are close enough together it could work, especially if the cloth is treated as a heavy canvas of some sort. Maybe a wood floor made out of another foldable wood frame could work instead if this doesn't.

Reply:
" Maybe a collapsible wood floor made out of another foldable wood frame could work instead if this doesn't." I think that would work better.  The more I think on this to be workable the supports would have to be so close you might as well use a foldable wooden frame.

Reply:
Fair enough, I went ahead and reworked the floor to be wooden, then.